driving_log_replayer/scripts/perception_evaluator_node.py

```diff
@@ -299,6 +280,15 @@ def __init__(self):
         )  # wall timer
         self.__skip_counter = 0
 
+    def get_frame_id_and_msg_type(self, evaluation_task: str):
```

Comment:
[NITS] It might be easier to understand using type hinting. If this is redundant, it would be good to define a type alias like `PerceptionMsgType = Union[DetectedObjects, TrackedObjects]`.

```suggestion
    def get_frame_id_and_msg_type(self, evaluation_task: str) -> Tuple[str, Union[DetectedObjects, TrackedObjects]]:
```